en_us.lang

```diff
@@ -157,200 +157,200 @@ gui.xaero_transfer_all=Transfer All Waypoints
+gui.xaero_toggle_waypoints=Toggle In-Game Waypoints
+gui.xaero_dots_scale=Entity Dot Scale
 gui.xaero_entity_info=Entity Info
 gui.xaero_toggle_slime=Toggle Slime Chunks
 gui.xaero_toggle_grid=Toggle Chunk Grid
 gui.xaero_compass_over_wp=Compass Over Waypoints
 gui.xaero_display_hostile=Display Hostile Mobs
-gui.xaero_box_entity_info=An interface displaying information about the living entity that you are pointing at including players.
+gui.xaero_box_entity_info=An interface displaying information about the living entity that you are pointing at, including players.
 gui.xaero_entity_info_settings=Entity Info Settings
-gui.xaero_entity_info_stay=Keep Entity Info For
+gui.xaero_entity_info_stay=Continue Displaying For
 gui.xaero_entity_info_distance=Reach Distance
-gui.xaero_entity_info_max_hearts=Max Amount of Hearts
-gui.xaero_current_biome=Show Current Biome
+gui.xaero_entity_info_max_hearts=Max Heart Number
+gui.xaero_current_biome=Display Current Biome
 gui.xaero_auto_map_size=Auto
-gui.xaero_reset_message=Are you sure you would like to reset default settings?
-gui.xaero_reset_message2=This action can't be reversed.
-gui.xaero_entity_info_numbers=Health In Numbers
+gui.xaero_reset_message=Are you sure you would like to restore the default settings?
+gui.xaero_reset_message2=This action cannot be undone!
+gui.xaero_entity_info_numbers=Health in Numbers
 gui.xaero_yaw=yaw
 gui.xaero_true=true
 gui.xaero_false=false
-gui.xaero_team_colours=Team Colours
+gui.xaero_team_colours=Team Colors
 gui.xaero_instant_waypoint=Quick Waypoint
 gui.xaero_entity_depth=Show Entity Depth
-gui.xaero_show_full_amount=Held Item Full Amount
-gui.xaero_show_flowers=Display Flowers
+gui.xaero_show_full_amount=Full Held Item Quantity
+gui.xaero_show_flowers=Show Flowers
 gui.xaero_show_entity_model=Display Entity Model
 gui.xaero_waypoint_names=WP Name Above Distance
-gui.xaero_waypoints_distance=Max WP Draw Dist.
+gui.xaero_waypoints_distance=Max WP Render Dist.
 gui.xaero_teleport_default_command=Default Teleport Command
-gui.xaero_waypoints_distance_min=Min WP Draw Dist.
+gui.xaero_waypoints_distance_min=Min WP Render Dist.
 gui.xaero_other_teams=Other Teams
-gui.xaero_entity_info_armour_numbers=Armour In Numbers
-gui.xaero_entity_info_armour=Armour Value
+gui.xaero_entity_info_armour_numbers=Armor in Numbers
+gui.xaero_entity_info_armour=Display Armor Info
 gui.xaero_arrow_scale=Arrow Scale
-gui.xaero_arrow_colour=Arrow Colour
-gui.xaero_smooth_dots=Smooth Dots
-gui.xaero_cave_zoom=Caves Zoom-in
-gui.xaero_lock_player_heads=Lock Player Heads
-gui.xaero_safe_mode_box=Use Java (CPU) based equivalent of the mod instead of OpenGL (GPU). Safe mode is a plan B, incase the mod fails normally. Not all features work in safe mode.
+gui.xaero_arrow_colour=Arrow Color
+gui.xaero_smooth_dots=Smooth Dot Edges
+gui.xaero_cave_zoom=Cave Mode Zoom In
+gui.xaero_lock_player_heads=Always Display Player Heads
```

Comment:
When will you increase the button width? If in the near future, then we can wait with this PR merger until then, but if not in the near future, then I can either shorten the button labels as you suggested or split this PR into two parts — I'll move the lines that exceed beyond buttons to the next PR (and put it on hold), and leave here only the non-problematic ones, so you will be able to merge without any blockers.

Reply:
I'm not sure it will fit with increased button size either. You can keep it like this for now. I'll probably use your changes on the same update as the button increase.

Reply:
If you plan to increase the button width to make the edge case in the default U.S. English translation fit, then the label above should fit as well. Comparison:
![Button-label-width-comparison](https://user-images.githubusercontent.com/34160838/93669170-5e26ed00-fa92-11ea-8556-5b1ab59e62d2.png)

Reply:
As expected, now since version 20.24.0 it looks fine.

Reply:
Well, I wasn't promising that "Ignore Server Heightmaps: World Map" would fit the new width either. Should be fine now though.

Reply:
Ah, okay, I had to overlook it then :/